### ISSUE TITLE
Integrating a new Re-ID model

### ISSUE THREAD

**Issue by alsonjiang** [ZQPei/deep_sort_pytorch#131]
Hi, how can I use a different Re-ID model to improve ID accuracy?

**Comment by Nishzzz** [comment]
Yeah. I'm also working on replacing the custom CNN used to extract features with other Reid models. If the author could please reply, it would be really helpful.

**Comment by pvtien96** [comment]
@Nishzzz please tell me if you have any updates/ related works. I'm also working on a similar project.

**Comment by SuperbTUM** [comment]
I am thinking of the embedding dim for the Reid model. Since the mainstream model is ResNet-50, is it okay to have a 2048-dim vector as feature? (However we only have 751 classes in Market1501)
